azavea/ism-watershed-wellness-snapshot
# Create development environment

## Overview

Sets up a development environment based on create-react-app, adds support for SASS, and other components of our typical project setup. Also demos that Redux is wired up correctly by implementing the counter demo from the app-template.

Connects #6 

### Demo

![dec-11-2018 15-08-58](https://user-images.githubusercontent.com/1042475/49827193-bf1a8080-fd56-11e8-86e1-e85482aaee63.gif)

### Notes

Follow-up issues: #7, #8 

## Testing Instructions

- Run `./scripts/setup`.
- Run `./scripts/server`.
- Visit the project in the browser.
- Verify that clicking the button increases the counter via Redux.
